source/extensions/filters/listener/proxy_protocol/proxy_protocol.cc

```diff
@@ -61,6 +62,13 @@ const KeyValuePair* Config::isTlvTypeNeeded(uint8_t type) const {
   return nullptr;
 }
 
+bool Config::isPassThroughTlvTypeNeeded(uint8_t type) const {
+  // TODO: read config.
+  if (type == 0)
+    return true;
```

Comment:
nit: to suppress the unused variable error.

Reply:
I think you can also change line 65 to be just "(uint8_t)"?  It's been a while since I've needed to do this =)